FFXIVWeather.Lumina/FFXIVWeatherLuminaService.cs

```diff
@@ -71,14 +71,14 @@ private Weather GetWeather(WeatherRate weatherRateIndex, int target)
             // Every zone has at least one target at 100, and weatherTarget's domain is [0,99].
             var rateAccumulator = 0;
             var weatherId = -1;
-            for (var i = 0; i < weatherRateIndex.UnkData0.Length; i++)
+            for (var i = 0; i < weatherRateIndex.Rate.Count; i++)
             {
-                var w = weatherRateIndex.UnkData0[i];
+                // var w = weatherRateIndex.UnkData0[i];
 
-                rateAccumulator += w.Rate;
+                rateAccumulator += weatherRateIndex.Rate[i];
```

Comment:
Think this is fine, it just turned from an array of structs of `Rate, Weather` into two separate arrays. This looks right